addons/mail/models/mail_alias.py

```diff
@@ -229,11 +229,22 @@ def open_parent_document(self):
         }
 
     def _get_alias_bounced_body_fallback(self, message_dict):
-        return Markup(
-            _("""<p>Hi,<br/>
-The following email sent to %s cannot be accepted because this is a private email address.
-Only allowed people can contact us at this address.</p>""")
-        ) % self.display_name
+        if self.alias_contact == 'partners':
+            group_description = _('addresses linked to registered partners')
+        elif self.alias_contact == 'followers':
+            group_description = _('a specific list of email addresses')
+        else:
+            group_description = _('addresses linked to registered employees')
+        default_email = self.env.user.company_id.email_formatted or self.env.user.email_formatted
+        return Markup(_("""<p>Dear Sender,<br /><br />
+The message below could not be accepted by the address %(alias_display_name)s.
+Only %(group_description)s are allowed to contact it.<br /><br />
+Please make sure you are using the correct address or contact us at %(default_email)s instead.<br /><br />
+Kind Regards,</p>""")) % {
+            'alias_display_name': self.display_name,
+            'group_description': group_description,
+            'default_email': default_email,
```

Comment:
Could be inlined no?

Reply:
It did not work, but then suddenly it did work.